packages/website/versioned_sidebars/version-latest-sidebars.json

```diff
@@ -21,15 +21,15 @@
       "items": [
         {
           "type": "category",
-          "label": "TypeSpec for HTTP",
+          "label": "TypeSpec for REST",
           "link": {
             "type": "doc",
-            "id": "getting-started/getting-started-http/index"
+            "id": "getting-started/getting-started-rest/index"
```

Comment:
I think this will crash the website build as this doesn't exist in the latest docs(either revert this file or update the latest docs as well)

Reply:
Good catch, thank you. Change reverted.